Intermediate Level 📁/kumar007ambi/data.json

```diff
@@ -0,0 +1,10 @@
+{
+  "user-details" : {
+    "user-name": "kumar007ambi",
+    "name": "Ambika Kumar Kewat",
+    "social-media": {
+      "linkedin": "https://www.linkedin.com/in/connect-with-ambika/",
+      "leetcode": "https://leetcode.com/u/Kum007/"
+    }
+  }
```

Comment:
We do not want your other profile , follow the rules.

Reply:
@Kushal997-das I have made required changes according to rule